interpretation/tests/test_train_pipeline.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
Can we put these in https://github.com/microsoft/seismic-deeplearning/blob/83efdc55298ca02977c5e482b72a700e4358a925/tests/cicd/aml_build.yml#L53

Reply:
@olgaliak should this be a follow up task so I don't work over the other open PR?

Reply:
Follow up for maxkazmsft: create AML Workspace in same sub and RG as the build VM, use those credentials here

Reply:
Use STANDARD_NC6 compute SKU

Reply:
Added guidance to activate seismic-interpretation in this commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/8f00605fa2de8efec29c4c061b8974efc7d9f8bf Added compute target guidance in this commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/f2269c517f6e25c06999ac06be228af73e50158f